### ISSUE TITLE
Why 1x, 2x-Schedule steps at 8,11 & 16, 22 Epoch

### ISSUE THREAD

**Issue by yc0619** [open-mmlab/mmdetection#5296]
Hi,

i have to write my master thesis on the theme. But i have no idea about, why is the step lr 1x 2x in such hyperparameter setted.
could someone help?

thanks!

**Comment by shinya7y** [comment]
[FPN](https://openaccess.thecvf.com/content_cvpr_2017/papers/Lin_Feature_Pyramid_Networks_CVPR_2017_paper.pdf
) uses a simple schedule: [60k, 80k] iterations (I write schedules including termination).
[Mask R-CNN (ICCV 2017)](https://openaccess.thecvf.com/content_ICCV_2017/papers/He_Mask_R-CNN_ICCV_2017_paper.pdf) adds one more step for human pose estimation: [60k, 80k, 90k] iterations.
[Detectron](https://github.com/facebookresearch/Detectron/blob/master/MODEL_ZOO.md#training-schedules) popularizes the schedule.
> - **1x**: For minibatch size 16, this schedule starts at a LR of 0.02 and is decreased by a factor of * 0.1 after 60k and 80k iterations and finally terminates at 90k iterations. This schedules results in 12.17 epochs over the 118,287 images in `coco_2014_train` union `coco_2014_valminusminival` (or equivalently, `coco_2017_train`).
> - **2x**: Twice as long as the 1x schedule with the LR change points scaled proportionally.

[Mask R-CNN (arXiv v3)](https://arxiv.org/abs/1703.06870v3) uses the 2x schedule as the updated baseline for instance segmentation and object detection.

MMDetection basically uses epoch-based training.
[60k, 80k, 90k] iterations correspond to [8, 11, 12] epochs.
`60000*16/118287 = 8.1`
`80000*16/118287 = 10.8`
`90000*16/118287 = 12.2`
